index.html

```diff
@@ -7,68 +7,113 @@
+                </main>
+                <aside class="contact-education-skills-box">
+                    <section class="contact-box">
+                        <h4 class="contact-title">Contact</h4>
+                        <section class="phone-box">
+                            <div class="phone-icon"><img src="phone-icon.svg" alt="phone"></div>
+                            <div class="phone"><p class="phone-details">(314) 174-7754</p></div>
+                        </section>
+                        <section class="mail-box">
+                            <div class="mail-icon"><img src="mail-icon.svg" alt="mail"></div>
+                            <div class="mail"><p class="mail-details">john.maverick@gmail.com</p></div>
+                        </section>
+                        <section class="linkedin-box">
+                            <div class="linkedin-icon"><img src="linkedin-icon.svg" alt="linkedin"></div>
+                            <div class="linkedin"><p class="linkedin-details">linkedin.com/in/john-maverick/</p></div>
+                        </section>
+                    </section>
+                    <section class="education-box">
+                        <div class="education">
+                            <h4 class="education-title">Education</h4>
+                            <span class="education-details">
+                                May 2017<br><span class="education-item-details">Georgia Institute of Technology,<br>Atlanta, GA</span><br>
+                                Bachelor of Science in Software Development<br>
+                                (GPA: 3.7/4.0)
+                            </span>
+                        </div>
+                    </section>
+                    <section class="relevant-skills-box">
+                        <div class="relevant">
+                            <h4 class="relevant-skills-title">Relevant Skills</h4>
+                            <p class=" relevant-skills-item">JavaScript</p>
+                            <p class=" relevant-skills-item">HTML5 & CSS3</p>
+                            <p class=" relevant-skills-item">JSON</p>
+                            <p class=" relevant-skills-item">ReactJS</p>
+                            <p class=" relevant-skills-item">Node.js</p>
+                            <p class=" relevant-skills-item">Express</p>
+                            <p class=" relevant-skills-item">SQL</p>
+                            <p class=" relevant-skills-item">MongoDB</p>
+                            <p class=" relevant-skills-item">Bootstrap</p>
+                            <p class=" relevant-skills-item">jQuery</p>
+                        </div>
+                    </section>
+                </aside>
+            </section>
+            <footer class="footer-box">
+                <span class="footer-text">I agree to the processing of personal data provided in this document for realising the recruitment process pursuant
```

Comment:
"I agree to the processing of personal data provided..." should begin in the next line.
```
<span class="footer-text">
  I agree to the processing of personal data...
  ...
</span>
```